packages/web-api/src/WebClient.ts

```diff
@@ -64,10 +64,6 @@ export enum WebClientEvent {
   RATE_LIMITED = 'rate_limited',
 }
 
-export interface WebAPICallOptions {
```

Comment:
Removing this generic interface, which is the basis for _all_ API request arguments.

Reply:
This is great! but to proceed with this, we may need to review all the argument types to verify if they have latest full list. Have you already done?

Reply:
I have not, and wanted to get feedback on this point; I mentioned in my original post that this change is significant enough that maybe it requires testing all the endpoints. It sounds to me like you agree that testing is needed. So, I will proceed with that.